scipy/stats/_distribution_infrastructure.py

```diff
@@ -0,0 +1,5082 @@
+        a random variable :math:`X` is:
+
+        .. math::
+
+            \mu'_n(X) = \int_{\chi} x^n f(x) dx
+
+        The "central" moment is the raw moment taken about the mean,
+        :math:`\mu = \mu'_1`:
+
+        .. math::
+
+            \mu_n(X) = \int_{\chi} (x - \mu) ^n f(x) dx
+
+        The "standardized" moment is the central moment normalized by a power of
+        the standard deviation, :math:`\sigma = \sqrt{\mu'_2}`:
+
+        .. math::
+
+            \tilde{\mu}_n(X) = \frac{\mu_n(X)}
+                                    {\sigma^n}
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'general', 'transform', 'normalize', 'quadrature', 'cache'}
+            The strategy used to evaluate the moment. By default (``None``),
+            the infrastructure chooses between the following options.
+
+            - ``'formula'``: use a formula for the differential entropy itself
+            - ``'general'``: use a general result that is true for all distributions
+                             with finite moments; for instance, the zeroth raw moment
+                             is identically 1.
+            - ``'transform'``: transform a raw moment to a central moment or
+                               vice versa; see notes.
+            - ``'normalize'``: normalize a central moment to get a standardized
+                               or vice versa.
+            - ``'quadrature'``: numerically integrate according to the definition
+            - ``'cache'``: use the value of the moment most recently calculated
+                           via another method.
+
+            Not all `method` options are available for orders, kinds, and
+            distributions. If the selected `method` is not available, a
+            `NotImplementedError`` will be raised.
+
+        Returns
+        -------
+        out : Array
+            The moment of the random variable of the specified order and kind.
+
+        See Also
+        --------
+        ContinuousDistribution.pdf
+        ContinuousDistribution.mean
+        ContinuousDistribution.variance
+        ContinuousDistribution.standard_deviation
+        ContinuousDistribution.skewness
+        ContinuousDistribution.kurtosis
+
+        Notes
+        -----
+        Not all distributions have finite moments of all orders; moments of some
+        orders may be undefined or infinite. If a formula for the moment is not
+        specifically implemented for the chosen distribution, SciPy will attempt
+        to compute the moment via a generic method, which may yield a finite
+        result where none exists. This is not a critical bug, but an opportunity
+        for an enhancement.
+
+        The definition of a raw moment in the summary is specific to the raw moment
+        about the origin. The raw moment about any point :math:`a` is:
+
+        .. math::
+
+            E[(x-b)^n] = \int_{\chi} (x-b)^n f(x) dx
```

Comment:
it is common to use `X` for a random variable, `x` is a specific realization of `X` (e.g., if you condition on `X=x`)

Reply:
Yes. This convention is used throughout the documentation. This is a typo.

Reply:
can be resolved